.gitignore

```diff
@@ -20,20 +16,13 @@ doc/build/*
 dist/
 .idea/
 playbooks/*.retry
-
-# Data Files
 *.csv
 *.db
-
-# Elastic Beanstalk Files
-.elasticbeanstalk/*
-!.elasticbeanstalk/*.cfg.yml
-!.elasticbeanstalk/*.global.yml
-
 .vagrant/
 Vagrantfile.local
 aws.yml.local
 devopsloft_workspace.code-workspace
 aws_ubuntu.pem
 *~
 .vscode/
+.secrets.json
```

Comment:
As discussed in the original PR, this is redundant

Reply:
Not sure I agree now. I have such a file and I don't want git tracking it.

Reply:
But there is already the `.secret*` in the gitignore...

Reply:
+1